### DISCUSSION TITLE
Unable to verify my academic affiliation

### DISCUSSION THREAD

**Discussion by fx999dev** [community/community#51131] [GitHub Education]
Hi,

I cant get to verify my academic affiliation. I did upload a certifacte of study, with todays date, which shows that I am a student until September 2024. I also added a short biography to my profile. My university is correctly recognised by Github. I don't know what else I can do now.

**Comment by SmashedFrenzy16** [comment]
Hi @fx999dev 

I have heard this issue pop up for many users on GitHub. As for trying to solve the issue, here is the following step that you can take:

1. When taking a picture of your ID, make sure the language is in English. Make sure that it is also very clearly visible after uploading.

2. If the above did not work, try contacting your university with the issue. They might be able to give you ID that works with GitHub

3. If the above has also not worked, try contacting [GitHub Support](https://support.github.com/).

Hope this helps and have a great day!

**Comment by Ntshangase** [comment]
Hello,

It will be easier to get assistance if you provide more information regarding your registration, Like a screenshot of the rejection reason and tell us what type of documents you submitted. 

Just a Tip: make sure that your browser is not blocking your **location** as that can also be the problems when applying.

**Comment by samus-aran** [comment]
Hi @fx999dev 

How did you get on? Did you do anything differently in your next submission?
